test/index.ts

```diff
@@ -140,6 +142,18 @@ describe('fp-ts', () => {
     assert.deepEqual(T.encode(new Set([])), [])
     assert.deepEqual(T.encode(new Set()), [])
   })
+
+  it('createStrMapFromDictionary', () => {
+    const T = createStrMapFromDictionary(t.number)
+
+    assert.deepEqual(T.decode({}), right(new StrMap({})))
+    assert.deepEqual(T.decode({ foo: 42 }), right(new StrMap({ foo: 42 })))
```

Comment:
Could you please add a failing test here?

Reply:
I hope the failing case is enough cause or should I at least check the `value`? The context object is really hard to test.

Reply:
Nevermind found your `PathReporter`.

Reply:
> The context object is really hard to test

You can leverage `PathReporter`, check out the test above (`createSetFromArray`)